server/middleware/validation.js

```diff
@@ -16,7 +16,10 @@ const validationErrors = {
     isValidWithinDate: label => `${label} must be within the last ${VALID_DAYS_RANGE} days`,
     validCaseReference: () => 'Case reference is not valid',
     contributionsFulfilled: () => 'Case contributions have to be completed or cancelled',
-    oneOf: () => 'Options are not valid'
+    oneOf: () => 'Options are not valid',
+    isValidMonth: () => 'Invalid month entered',
+    isValidYear: () => 'Invalid year entered',
+    isValidDay: () => 'Invalid day entered'
```

Comment:
You [shouldn't](https://design-system.service.gov.uk/components/error-message/#be-clear-and-concise) use "invalid" here because it doesn't add anything to the message. A good error message helps the user understand what went wong and how to fix it.

There's guidance on the "right" error messages for date validation -- see https://design-system.service.gov.uk/components/date-input/#if-the-date-entered-cannot-be-correct.

Reply:
Updated and followed the "If the date entered cannot be correct" example as closely as possible. Ideally I think we would have a common message for all of these and use highlighting for more information as specified but due to the way the validator code is shared system wide I think changing it to enable this would have far reaching implications and not be worth it at this point